### ISSUE TITLE
Multi-line statements in the shell maybe?

### ISSUE THREAD

**Issue by RosiePuddles** [fck-language/fck#96]
### The problem
I'm trying to change how code is run, so that if you give it a block of code it works out the AST for the whole thing before interpreting it. So I wanted to test out how it handled single line statements with a `;` in the middle to emulate a multi-line statement like this:
```
>>> int a :: 5;float b
```
which returned this error:
```Python
>>> int a :: 5;float b
Traceback (most recent call last):
  File "/Users/$USER/Documents/Code/.fck/source/fck/PythonShell/cli.py", line 51, in <module>
    shell()
  File "/Users/$USER/Documents/Code/.fck/source/fck/PythonShell/shell.py", line 25, in shell
    res = run('<shell>', text, previous)
  File "/Users/$USER/Documents/Code/.fck/source/fck/PythonShell/fck_main.py", line 2071, in run
    ast = parser.parse()
  File "/Users/$USER/Documents/Code/.fck/source/fck/PythonShell/fck_main.py", line 323, in parse
    res = self.statements()
  File "/Users/$USER/Documents/Code/.fck/source/fck/PythonShell/fck_main.py", line 359, in statements
    statement = res.try_register(self.statement())
  File "/Users/$USER/Documents/Code/.fck/source/fck/PythonShell/fck_main.py", line 429, in statement
    expr = res.register(self.expr())
  File "/Users/$USER/Documents/Code/.fck/source/fck/PythonShell/fck_main.py", line 461, in expr
    pos_start, expr.pos_end))
AttributeError: 'NoneType' object has no attribute 'pos_end'
```

So I guess it can't handle multi-line statements

### What did expect...
The following two are supposed to be equivalent:
```
>>> int a :: 5;float b
```
```
>>> int a :: 5
>>> float b
```
which at the moment they are not

### Informations (please complete the following information):
 - OS: iOS Big Sur (v11.5.2) M1 chip
 - Branch: main

### Other stuff
Normally I wouldn't do this for a project that was ending so soon but I want to be able to have a working prototype to see how it might work would be **REALLY** useful for transition into Rust so I just hope it'll be worth it

**Comment by RosiePuddles** [comment]
This issue wasn't anything to do with the fact that the input was a multiline input. It was a problem to do with default value assignments, fixed in 3aad539835516933f63b0f1a3342b8cd0daa7d17.
This fix has now resulted in the input `int a :: 5; float b` returning `[None, None]`, which I mean yes, but also not really what I wanted
<details>
<summary>Script in context</summary>

```
>>> int a :: 5; float b
[None, None]
>>> 
```
</details>

To fix this is more complex, and has roots in what `Parser.parse()` returns (`List` type). The `Parser.parse()` method should return a `list` type, the Python primitive not the fck primitive. This list should be a list of ASTs that should then be executed one by one by the Interpreter.

### Shell mode
If the interpreter is running in shell mode, the output of each AST should be printed to the shell, unless the value is `None` in which case it should be ignored

**Comment by RosiePuddles** [comment]
Closed due to issue relating to Python
